gensim/summarization/textcleaner.py

```diff
@@ -3,6 +3,13 @@
 #
 # Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
 
+"""Text Cleaner
+
+This module contains functions and processors used for processing text, 
+extracting sentences from text, working with acronyms and abbreviations.
```

Comment:
Need to add examples/highlights/motivation here (after you finish with docstrings in this file).